data_generators/user_generators.py

```diff
@@ -50,8 +51,9 @@ def __init__(self):
         self.low_income = percent_true(70)
         self.list_convictions = sample(convictions, k=randint(1, 3))
```

Comment:
Change this to be a string not a list.

Reply:
updated list_convictions to the first object in the list, which should be a string.